client/index.html.tpl

```diff
@@ -32,6 +32,7 @@
 	<body class="signed-out<%- public ? " public" : "" %>" data-transports="<%- JSON.stringify(transports) %>">
 		<div id="viewport" role="tablist">
 			<aside id="sidebar">
+				<img src="img/logo-horizontal-transparent-bg-inverted.svg" class="logo" alt="The Lounge" height="45">
```

Comment:
This logo isn't included in this commit.

Reply:
Also wrap the logo in a div.

Reply:
🤦‍♂️
Of course I forgot to include it.

Alright, I have pushed it, but it's quickly as I have 5 minutes on the laptop. It's not properly minified, no non-inverted version not wrapped in a div, etc.
I'll look and address other comments later :)

Reply:
> Also wrap the logo in a div.

@xPaw, you sure? As far as I can tell, we're not doing this for other logos (loading and sign in pages).

Reply:
Wrapping is gonna allow for better styling (e.g. a background). It's more reasonble on the sidebar because there's not much you can do with it on the loading/sign in screens.